### PR TITLE
Add bombastic indexer

### PR DESCRIPTION
**Pull request by gildub**

### CODE REVIEW

File: roles/tpa_single_node/templates/manifests/bombastic/indexer/Deployment.yaml.j2 [hunk 1]
```diff
@@ -28,9 +28,15 @@ spec:
       volumes:
         - name: data
           emptyDir: {}
+        - name: v11yapisecret
```
Comment (desmax74) [inline roles/tpa_single_node/templates/manifests/bombastic/indexer/Deployment.yaml.j2]:
```suggestion
        - name: v11y-api-secret
```

File: roles/tpa_single_node/templates/manifests/bombastic/indexer/Deployment.yaml.j2 [hunk 2]
```diff
@@ -28,9 +28,15 @@ spec:
...
+          secret:
+            secretName: storage_secret
+        - name: v11yapisqssecret
```
Comment (desmax74) [inline roles/tpa_single_node/templates/manifests/bombastic/indexer/Deployment.yaml.j2]:
```suggestion
        - name: v11y-api-sqs-secret
```

File: roles/tpa_single_node/templates/manifests/bombastic/indexer/Deployment.yaml.j2 [hunk 3]
```diff
@@ -42,69 +48,92 @@ spec:
...
-            - name: data
-              mountPath: /data
           ports:
             - containerPort: 9010
               protocol: TCP
               name: infra
+          volumeMounts:
+            - name: data
+              mountPath: /data
+            - mountPath: /etc/v11yapisecret
```
Comment (desmax74) [inline roles/tpa_single_node/templates/manifests/bombastic/indexer/Deployment.yaml.j2]:
```suggestion
            - mountPath: /etc/v11y-api-secret
```

File: roles/tpa_single_node/templates/manifests/bombastic/indexer/Deployment.yaml.j2 [hunk 4]
```diff
@@ -42,69 +48,92 @@ spec:
...
+              name: v11yapisecret
+            - mountPath: /etc/v11yapisqssecret
```
Comment (desmax74) [inline roles/tpa_single_node/templates/manifests/bombastic/indexer/Deployment.yaml.j2]:
```suggestion
            - mountPath: /etc/v11y-api-sqs-secret
```

**Review by desmax74 — CHANGES_REQUESTED** [review]
LGTM , with just two minor name changes

**Review by mrrajan — APPROVED** [review]
LGTM
image

**Review by desmax74 — APPROVED** [review]

**Pull request merged**